src/Platforms/DB2Platform.php

```diff
@@ -634,6 +635,17 @@ public function getAlterTableSQL(TableDiff $diff)
                 continue;
             }
 
+            $newColumnName = $columnDiff->getNewColumn()->getQuotedName($this);
+            $oldColumn     = $columnDiff->getOldColumn() ?? $columnDiff->getOldColumnName();
+
+            $oldColumnName = $oldColumn->getQuotedName($this);
+
+            // Column names in DB2 are case insensitive and automatically uppercased on the server.
```

Comment:
So… if a user tries to change the case, we ignore them silently? Shouldn't we instead let them know there is no point in doing this?

Reply:
If we do a check, it should likely be during the `renameColumn` use directly, because from what I gathered all the availlable database drivers are case insensitive, and the problem I had was when a column is created with a lowercase column (eg: `c1`), some drivers (like Oracle and SQL server) actually creates it all uppercase (eg: `C1`) and then when computing the diff the `oldColumnName` is uppercase while the current column name is lowercase even though the user didn't actually request for a change but it was computed automatically

It's a broader issue than this PR can tackle but I do think the column name normalization should maybe take care of this, or maybe when during the diff computing beforehand